nano/secure/ledger.hpp

```diff
@@ -67,6 +75,7 @@ class ledger final
 	std::atomic<size_t> bootstrap_weights_size{ 0 };
 	uint64_t bootstrap_weight_max_blocks{ 1 };
 	std::atomic<bool> check_bootstrap_weights;
+	std::atomic<bool> enable_pruning{ false };
```

Comment:
Does it need too be atomic? Is it not always modified in single thread (node init) or tests?

Reply:
Maybe short-handing to just `pruning` as well?

Reply:
Updated